src/legacy/core_plugins/data/public/query/query_bar/components/__snapshots__/query_string_input.test.tsx.snap

```diff
@@ -1907,14 +1892,9 @@ exports[`QueryStringInput Should pass the query language to the language switche
                     },
                     "http": Object {
                       "addLoadingCount": [MockFunction],
-                      "anonymousPaths": AnonymousPaths {
-                        "basePath": BasePath {
-                          "basePath": "",
-                          "get": [Function],
-                          "prepend": [Function],
-                          "remove": [Function],
-                        },
-                        "paths": Set {},
+                      "anonymousPaths": Object {
+                        "isAnonymous": [MockFunction],
+                        "register": [MockFunction],
```

Comment:
@elastic/kibana-app-arch I updated an implementation in mocks and got false positive failure. Does it mean that we have to update tests whenever public interfaces changed?

Reply:
No, I don't think React component tests should snap mocks of core.

Reply:
Let's merge it and you can update the tests after